cmd/showref.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/driusan/dgit/git"
+)
+
+func ShowRef(c *git.Client, args []string) error {
```

Comment:
I'm not sure how I feel about having this lie about being successful just to get `go get` to not report an error, but I think I might need to implement `show-ref` to `git upload-pack` to work. Can you have this parse the options listed in the man page into a struct and I'll pick it up from there?

Reply:
Sure, I can do that.

For now, the rationale is that it doesn't support any refs and shows nothing. It's certainly a stretch, but it successfully shows nothing. :)